src/plugins/discourse/references.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+const htmlparser2 = require("htmlparser2");
```

Comment:
`htmlparser2` claims to be geared towards node. And needs browserify to work elsewhere.
Am I right in assuming web clients would never use this plugin directly because it's incorporated into the graph?

Reply:
Yep, I don't expect web clients to depend on this directly. If we wind up needing it in the frontend, we can change deps (usage of htmlparser2 is very local)